packages/just-scripts/src/stack/getAvailableStacks.ts

```diff
@@ -0,0 +1,27 @@
+import path from 'path';
+import { readPackageJson } from 'just-scripts-utils';
+
+export function getAvailableStacks(rootPath: string) {
+  const scriptsPath = path.join(rootPath, 'scripts');
+  const packageJson = readPackageJson(path.join(rootPath, 'scripts'));
+
+  if (!packageJson) {
+    throw new Error(`not able to read package.json from ${scriptsPath}`);
+  }
+
+  const devDeps = packageJson.devDependencies || {};
+
+  const stackDeps = Object.keys(devDeps).reduce<{ [key: string]: string }>((collected, dep) => {
```

Comment:
I think this would be clearer if done without reduce (just create the object ahead of time, do a for loop, and modify it each iteration). Might be just me but I find most things using reduce to be hard to understand at a glance.

Reply:
You've mentioned this... it's a _just_ reducer! But I get what you mean.